src/main/java/com/revature/controllers/ContentController.java

```diff
@@ -103,21 +103,12 @@
 	@GetMapping(params = { "title", "format", "modules" })
 	public ResponseEntity<Set<JSONContent>> getSearchResults(
 			@RequestParam(value = "title", required = false) String title,
-			@RequestParam(value = "format", required = false) String format,
-			@RequestParam(value = "modules", required = false) String modules) {
-		ArrayList<Integer> moduleIdsList = new ArrayList<Integer>();
-		ArrayList<String> formatList = new ArrayList<String>();
-		StringTokenizer st = new StringTokenizer(modules, ",");
-		while(st.hasMoreTokens()) {
-			moduleIdsList.add(Integer.parseInt(st.nextToken()));
-		}
+			@RequestParam(value = "format", required = false) List<String> formats,
+			@RequestParam(value = "modules", required = false) List<Integer> modules) {
 		
-		StringTokenizer ft = new StringTokenizer(format, ",");
-		while (ft.hasMoreTokens()) {
-			formatList.add(ft.nextToken());
-		}
-
-		Set<Content> contentSet = searchService.filter(title, formatList, moduleIdsList);
+		System.out.println(formats.toString());
+		System.out.println(modules.toString());
```

Comment:
remove or replace with logger

Reply:
Sorry Carolyn, didnt mean for you to see this, these are for troubleshooting and will be removed.  I was making sure the spring refactor was working and noticed faulty behavior in the module search.  Currently diving into it more but will let you know after the investigation is done.

Reply:
Sysouts have been removed.